### PR TITLE
Update prettier to v2 to support new TypeScript syntax

### LINKED ISSUE #150: Support type only imports in TypeScript 3.8
TypeScript 3.8 release notes: https://devblogs.microsoft.com/typescript/announcing-typescript-3-8/

### PR DESCRIPTION
**Pull request by aminya**
Fixes https://github.com/Khan/flow-to-ts/issues/150

- This updates prettier to v2 which allows using newer typescript such as `import type`. 
- The files had to be reformated by prettier to allow the tests to pass
- Noe 8 was removed from the CI which is not supported by Prettier v2 anymore or Node itself

### CODE REVIEW

**Comment by k4b7** [comment]
@aminya thanks for the PR! You'll need to sign our Contributor License Agreement before I can accept this contribution. Please comment here once you've done so.

**Comment by aminya** [comment]
The document is signed!

File: package.json [hunk 1]
```diff
@@ -44,7 +44,8 @@
   "scripts": {
     "test": "jest",
     "coverage": "jest --coverage --collectCoverageFrom=src/**/*.js --coveragePathIgnorePatterns=src/flow-to-ts.js",
-    "pretty-quick": "prettier src/*.js test/*.js playground/src/*.tsx babel-generator/lib/**/*.js babel-traverse/lib/**/*.js --write"
+    "pretty-quick": "prettier src/*.js test/*.js playground/src/*.tsx babel-traverse/lib/**/*.js --write",
+    "pretty-tests": "prettier ./test/fixtures/**/ts.js --parser typescript --write && prettier ./test/fixtures/**/flow.js --parser flow --write"
```
Comment (k4b7) [inline package.json:48]: `pretty-tests` should probably run as part of the pre-commit hook as well, but it's not critical for this PR.

**Review by k4b7 — APPROVED** [review]
LGTM.  I tried the preview playground and `import type {Foo} from "./foo"` is working there.  Thanks for updating prettier.

**Pull request merged**